### PR TITLE
Module contents skeletons

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**

### CODE REVIEW

File: d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js [hunk 1]
```diff
@@ -290,6 +290,10 @@ class D2LSequenceLauncherModule extends ASVFocusWithinMixin(PolymerASVLaunchMixi
 			_moduleWasExpanded: {
 				type: Boolean,
 				value: false
+			},
+			_skeleton: {
```
Comment (pbrown-d2l) [inline d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: NIT: Should be _skellington

Comment (ChrisKraljevicD2L) [reply d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: skellington

**Comment by ianlaidlaw** [comment]
screenshots / gifs ?

File: d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js (hunk 1 above)
Comment (ianlaidlaw) [reply d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: serious nit though, if it's a boolean i would name it something like `_showSkeleton` or `_shouldShowSkeleton` or `showLoadingSkeleton` or something

Comment (ChrisKraljevicD2L) [reply d2l-sequence-launcher-unit/d2l-sequence-launcher-module.js]: Good idea, sorry for not getting screenshots/gif sooner I had to rebuild my instance.

**Comment by ChrisKraljevicD2L** [comment]
launcher_skeletons

**Comment by ianlaidlaw** [comment]
> LGTM after that boolean name change

Sorry I didn't refresh my page after you made changes

**Pull request merged**